locallib.php

```diff
@@ -483,6 +483,14 @@ public static function coursedisplaytext($course, $tpl) {
         return strip_tags($displaytext);
     }
 
+    /**
+     * Apply length limits to a template string. TOKEN{#} in the template string
+     * is replaced by TOKEN, and the replacement value for TOKEN is truncated to
+     * # characters.
+     *
+     * @param object $replacements an array of pattern => replacement
+     * @param string $tpl the template string (coursename or category)
+     */
     public static function apply_template_limits(&$replacements, &$template) {
```

Comment:
The string names in the docblock need to match the string names in the signature. It's also helpful to specify the `return`.

Reply:
Docblock param names fixed. This function does not return anything -- I think it makes more sense to leave out `@return`?